.gitignore

```diff
@@ -10,6 +10,5 @@
 .DS_Store
 /build
 /captures
-/app/google-services.json
```

Comment:
actually.. we should add this back..

we committed the default version., and we add this line back

Reply:
so that ppl can add theirs, but it won't be flag as diff so they won't check back theirs in..

I assume yours is 100% public now :D